app/routes/orders/contact_summary.js

```diff
@@ -1,10 +1,18 @@
-import detail from './detail';
+import detail from "./detail";
+import AjaxPromise from "stock/utils/ajax-promise"; //jshint ignore:line
 
 export default detail.extend({
-  setupController(controller, model){
-    if(controller) {
+  async setupController(controller, model) {
+    if (controller) {
       this._super(controller, model);
-      controller.set('isActiveSummary', true);
+      let userId = model.get("createdBy.id");
+      let ordersCount = await new AjaxPromise(
```

Comment:
`setupController` as name suggests should only be used to set data on controller. It shouldn't request data from API. Use `model` hook for that. Use nested promises if needed.